microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -3,254 +3,284 @@ description: The latest experimental features in Microsoft Edge DevTools
 
-The **Inspect** tool provides a quick way to identify and visualize CSS Flexbox layouts in a website by hovering on them with the mouse.  Choose the **Inspect** \(![Inspect](../media/inspect-icon.msft.png)\) icon in the top-left corner of DevTools.  Then, while debugging the website, hover on a flex container to display outlines around the flex container.  
+*   Switch between units for different font properties
+*   Switch between keywords for different font properties
+*   Convert units
+*   Generate accurate CSS code
 
-:::image type="complex" source="../media/flexbox-hover.msft.png" alt-text="Display Flexbox containers with the Inspect tool" lightbox="../media/flexbox-hover.msft.png":::
-   Display Flexbox containers with the **Inspect** tool  
-:::image-end:::  
+To use the new visual **Font Editor**:
+1.  Open the **Elements** tool.
+1.  Open the **Styles** pane.
+1.  Select the **Font Editor** icon.
 
-#### Display persistent overlays on Flexbox layouts  
+For more information about the new visual **Font Editor**, navigate to [Edit CSS font styles and settings in the Styles pane in DevTools][DevtoolsInspectStylesEditFonts].
 
-In Microsoft Edge version 89 or later, the experimental CSS Flexbox feature also offers the option to turn on persistent overlays on Flexbox layouts.  Persistent overlays provide the following benefits.  
+:::image type="complex" source="../media/font-editor-open.msft.png" alt-text="The visual Font Editor pane is highlighted" lightbox="../media/font-editor-open.msft.png":::
+   The visual **Font Editor** pane is highlighted
+:::image-end:::
 
-*   Persistent overlays remain visible on the webpage as you scroll, move your mouse, and use other features of the DevTools.
-*   Multiple persistent overlays may be used at the same time, to allow you to review several Flexbox layouts at once.  
-*   Persistent overlays offer color configuration options.  
-    
-To toggle persistent overlays on Flexbox layout, use one of following actions.  
-
-*   Choose the **Flexbox** oval icon next to any Flexbox container displayed in the DOM tree of the **Elements** tool.  
-*   Open the new **Layout** panel located in the **Elements** tool, and choose the checkbox next to each Flexbox container you want to highlight.  
-    
-:::image type="complex" source="../media/flexbox-overlay.msft.png" alt-text="Flex icons and Layout panel in DevTools" lightbox="../media/flexbox-overlay.msft.png":::
-   Flex icons and **Layout** panel in DevTools  
-:::image-end:::  
+This experiment is available starting with Microsoft Edge version 89.
 
-#### Configure persistent overlays  
 
-To configure options for persistent overlays for CSS grids or Flexbox layouts, use the **Layout** pane.  The **Layout** pane is located in the **Elements** tool next to the **Styles** and **Computed** panes.  
+<!-- ====================================================================== -->
+## Previously Experimental features which are now regular features
```

Comment:
Since those features are not experimental anymore, I don't think they should be on this page anymore. Instead, the content should move to other pages in the docs that deal with the associated topic already (or creating new pages if needed).
We should probably do this in a separate PR though, and keeping the content here in the meantime is a good idea. So no change needed here, but wanted to raise it to avoid forgetting.